src/Plug.vala

```diff
@@ -1,41 +1,38 @@
-// -*- Mode: vala; indent-tabs-mode: nil; tab-width: 4 -*-
-/*-
- * Copyright (c) 2013 Pantheon Developers (http://launchpad.net/online-accounts-plug)
- *
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Library General Public
- * License as published by the Free Software Foundation; either
- * version 2 of the License, or (at your option) any later version.
- *
- * This library is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Library General Public License for more details.
- *
- * You should have received a copy of the GNU Library General Public
- * License along with this library; if not, write to the
- * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
- * Boston, MA 02110-1301 USA.
- *
- * Authored by: Corentin Noël <tintou@mailoo.org>
- */
-namespace OnlineAccounts {
+/*
+* Copyright (c) 2013-2018 elementary, Inc. (https://elementary.io)
```

Comment:
one space is missing on every line to align all the asterisks, and remove the `(c)` as it is officially redundant with the Copyright word (see Linux foundation courses about Copyrights)